docs/faq.rst

```diff
@@ -557,6 +557,8 @@ These are your options to protect against that:
 - Mount C's filesystem on another machine and then create a backup of it.
 - Do not give C filesystem-level access to S.
 
+See :ref:`hosting_repositories` for a detailed protection guide.
```

Comment:
detailled - 2x L

Reply:
https://dict.leo.org/englisch-deutsch/detailed

Reply:
interesting. i was quite sure there is a rule that doubles the L, but otoh, English is not very regular...